public/components/event_analytics/explorer/events_views/data_grid.tsx

```diff
@@ -65,14 +70,18 @@ export function DataGrid(props: DataGridProps) {
     pplService,
     requestParams,
   });
+
+  const query = useSelector(selectQueries)[tabId];
+  const defaultTimestamp = query[SELECTED_TIMESTAMP];
+
   const selectedColumns =
     explorerFields.selectedFields.length > 0
       ? explorerFields.selectedFields
-      : DEFAULT_EMPTY_EXPLORER_FIELDS;
+      : [{ name: defaultTimestamp, type: 'timestamp' }, ...DEFAULT_EMPTY_EXPLORER_FIELDS];
```

Comment:
DEFAULT_EMPTY_EXPLORER_FIELDS has already contained ```{ name: 'timestamp', type: 'timestamp' }``` will this lead to selectedColumns to have:
```
selectedColumns = [
{ name: defaultTimestamp, type: 'timestamp' }
  { name: 'timestamp', type: 'timestamp' },
  { name: '_source', type: 'string' },
]
```
?

Reply:
Actually, I modified DEFAULT_EMPTY_EXPLORER_FIELDS to remove this line: `{ name: 'timestamp', type: 'timestamp' }`, because fundamentally there cannot a default timestamp name, as that is used to map the data to the columns.